File: p001/p1.lua
#! /usr/bin/env lua
-- Project Euler Problem #1

sum = 0

for i = 0, 1000-1 do
    if i%3 == 0 or i%5 == 0 then
    sum = sum + i
    end
end
print ('sum of all values below 1000 divisible by 3 or 5 is ' .. sum)   

File: p001/p1.py
#! /usr/bin/env python
# Project Euler Problem #1

if __name__ == "__main__":
    
    result  = sum([i for i in xrange(3,1000) if i%3 == 0 or i%5 == 0])

    print "The sum of all natural numbers <1000 and divisible by 3 or 5 is: %s" %result
